continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
+    run:
+      image:
+        from_service: web
+
+      commands:
+        - container setup
```

Comment:
I'm concerned now having this in the seeds may lead to unintentional things happening, due to `setup` meaning first run only. Some images are built like this